src/components/header/SocietyButtons.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PageHeader from '../header/PageHeader';
+import Buttons from '../buttons/SocietyTabButtons';
+
+/* eslint-disable */
+class SocietyButtons extends Component {
+ constructor(props){
+   super(props);
+   this.state= {
+     active: 'istelle'
+   }
+ }
+ setActiveButton = (filter) => {
+   return () =>{
+    this.setState({
+      active:filter
+    })
+   }
+ }
+ render() {
+   return (
+     <div className='societyButtons'>
+       <Buttons active={this.state.active} filter='istelle' setActiveButton={this.setActiveButton}>iStelle</Buttons>
```

Comment:
These buttons should just be one button that's reusable. The filter fields should be all in small caps coz its more or less being used as a variable.